tests/TestClassTemplateCall.php

```diff
@@ -55,6 +55,14 @@ class TestClassTemplateCall
 
 	/** @var array */
 	private $_expectationTemplatePath = [
+		1 => [
+			'path' => [
+				'...\wordpress\wp-content\plugins/onoffice-themes/templates/address/SearchFormAddress.php' => "SearchFormAddress.php",
```

Comment:
Where's that back-slash/forward-slash mix coming from?

Reply:
@jayay It comes from this function
https://github.com/onOfficeGmbH/oo-wp-plugin/blob/a789d6b1f81acffea9cdbec994a9325e3489fd34/plugin/Model/FormModelBuilder/FormModelBuilder.php#L116
Glob function will return the path separated by backslash

Reply:
I don't think this has any effect on this function